youthful-piety/modernized/03-chapter-02.adoc

```diff
@@ -244,55 +244,64 @@ in which case he would wait until he was left alone.
 It was with difficulty he could manage to write, yet he occasionally penned memorandums,
 a few of which are here inserted.
 
-"`Eighth month, 1836.--I have now begun to read the Scriptures regularly.
-I trust Providence will enable me to understand what I read.`"
+[.embedded-content-document]
+--
+
+Eighth month, 1836.--I have now begun to read the Scriptures regularly.
```

Comment:
Also, should the dates of these entries be treated differently? I thought that we might have done that before, but I couldn't remember how.